src/java.management/share/classes/java/lang/management/MemoryMXBean.java

```diff
@@ -267,6 +267,24 @@ public interface MemoryMXBean extends PlatformManagedObject {
      */
     public MemoryUsage getNonHeapMemoryUsage();
 
+    /**
+     * Returns the CPU time used by all garbage collection threads.
+     *
+     * <p> This include time since genesis, so the value can be
+     * non-zero even if no garbage collection cycles occured. In
+     * general this includes time for all driver threads,
+     * workers, VM operations on the VM thread and the string
+     * deduplication thread (if enabled). This method returns
+     * {@code -1} if the platform does not support this operation
+     * or if called during shutdown.
```

Comment:
Can we explicitly state here that this is concurrent time as well as worker time spent during pauses but not accumulated pause times?

Reply:
Just want to ensure that I read this correct, please correct any mistake below:

* concurrent time = `CPU time`
* accumulate times = `wall-clock time`

Is your suggestion that we should clarify that the method may only account for `CPU time` and not `wall-clock time` during pauses?

Reply:
"CPU time used by all garbage collection threads"  or even "Accumulated CPU time..." would seem clear that we mean these threads, in this process.  We can leave genesis out of it, whether it meant start of process or dawn of time. 8-)

Yes, might be better as just, "This may include some startup work and overhead, therefore the value can be non-zero even...".